app/routes/synonyms/route.tsx

```diff
@@ -10,12 +10,14 @@ import { Outlet } from '@remix-run/react'
 import { GridContainer } from '@trussworks/react-uswds'
 
 import { getUser } from '../_auth/user.server'
+import { feature } from '~/lib/env.server'
 
 export async function loader({ request }: LoaderFunctionArgs) {
   const user = await getUser(request)
   const isModerator = user?.groups.includes('gcn.nasa.gov/circular-moderator')
+  const featureFlagOn = feature('SYNONYMS')
 
-  if (!isModerator) {
+  if (!isModerator || !featureFlagOn) {
```

Comment:
```suggestion
  if (!isModerator || !feature('SYNONYMS')) {
```
No need to define as a const